src/components/MeasurementPlot.tsx

```diff
@@ -72,6 +72,8 @@ function MeasurementComponent(props: MeasurementPlotProps) {
   } = props;
   const { title = '', data } = measurement;
 
+  const xAxis = useMemo(() => `${xVariableName}-x`, [xVariableName]);
+  const yAxis = useMemo(() => `${yVariableName}-y`, [yVariableName]);
```

Comment:
Strings are always unique, you don't need to memo them.

Reply:
removed, thank you